### PR TITLE
Skip token of newline in getLocalReadWrite to avoid stoull(newline)

### PR DESCRIPTION
**Pull request by meahoibm**
Compiled in 8.4 docker, used local tools to create a repo and uninstall/install RPMS and start up daemons.  Ran against robot from Jenkins for testing.

### CODE REVIEW

**Comment by meahoibm** [comment]
Unit test of getdeviceusage;
[root@c650f06p33-ug meaho]# bbcmd getdeviceusage
{"id":"1","rc":"0","in":{"apicall":"BB_GetDeviceUsage","parms":{"device":"0"},"misc":{"uid":"0","gid":"0"}},"env":{"jobid":"1","jobstepid":"1","contribid":"999999998"},"out":{"critical_warning":"0","temperature":"40","available_spare":"100","percentage_used":"1","data_read":"1178416798","data_written":"1585097460","num_read_commands":"1185129904","num_write_commands":"1098480925","busy_time":"7914","power_cycles":"145","power_on_hours":"28213","unsafe_shutdowns":"104","media_errors":"0","num_err_log_entries":"0"}}
[root@c650f06p33-ug meaho]# 
Now going to run robot fvt

**Comment by meahoibm** [comment]
Reverted changes to getdeviceusage mapping but still look to strip off any tabs in the labels.

File: bb/src/usage.cc [hunk 1]
```diff
@@ -292,6 +300,7 @@ int proxy_GetDeviceUsage(uint32_t devicenum, BBDeviceUsage_t& usage)
         string name = tokens[0];
         string val  = tokens[1];
         size_t index;
+        name.erase(name.find_last_not_of("\t")+1);
```
Comment (tgooding) [inline bb/src/usage.cc:303]: Should this be find_last_not_of("\t ")?  Whereby excluding either a tab character or space character?

**Review by tgooding — APPROVED** [review]
lgtm

**Pull request merged**